packages/eslint-config-terra/package.json

```diff
@@ -39,5 +39,8 @@
   },
   "peerDependencies": {
     "eslint": "^6.1.0"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0"
```

Comment:
Should this be a hard dependency so it's provided when eslint-config-terra is installed? Does this lock us into a specific babel version?

Reply:
https://github.com/cerner/terra-toolkit/pull/424/files#diff-ac4371ccbae65851f05c94f13356f88eR33

arg good catch, yeah this should be a hard dependency so downstream projects pull it it. the parser expects a babel version of at least 7.2 via peer dependency https://github.com/babel/babel-eslint/blob/master/package.json